CONTRIBUTING.md

```diff
@@ -33,6 +35,14 @@ pytest tests/
 
 ## Contribution Guidelines
 
+### DCO and Signed-off-by
+
+When contributing changes to this project, you must agree to the [DCO](DCO).
+Commits must include a `Signed-off-by:` header which certifies agreement with
+the terms of the [DCO](DCO).
+
+Using `-s` with `git commit` will automtaically add this header.
```

Comment:
```suggestion
Using `-s` with `git commit` will automatically add this header.
```

Reply:
Maybe also add a link to https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits?

Reply:
Commit signing is different than the `Signed-off-by` bit. Signing with gpg isn't necessary to meet the requirement. (at least in every project I've worked on that uses the DCO)

Reply:
Oh, I got confused because I ran into this issue when trying to set up auto sign on my IDE.